cypress/utils/steps.js

```diff
@@ -239,8 +239,8 @@ function selectDropdownItem(label, item) {
   cy.get('@dropDownButton')
     .parent()
     .within(() => {
-      cy.get('@dropDownButton').click();
-      cy.contains('[role="menuitem"] span', item).click();
+      cy.get('@dropDownButton').click({force: true});
```

Comment:
Can you explain why this is needed?
Which test fails if we don't add it?

Reply:
The `Test Backend Simple Workflow - can edit an existing entry and publish` test fails without this.

I don't understand why this test fails.
I am confused as to why this test needs to be changed to work with my changes.

Reply:
As per this comment https://github.com/netlify/netlify-cms/pull/5595#issuecomment-888172829 this test is broken and can be ignored